### PR TITLE
feat: mutualize makefiles

### PR DESCRIPTION
**Pull request by pcarles**
# Description

Goal here is to generalize theses makefile to avoid code duplication accross our repositories

# Submission Checklist

- [x] The code author has performed a self-review of the code.
- [x] The code author has performed a complete test of the code.
- [x] The documentation is up to date.

### CODE REVIEW

File: makefiles/ansible.mk [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ssh-config:
+ifndef SSH_AUTH_SOCK
+	$(error Please load ssh-agent)
+endif
+
+install-roles: ssh-config
+	@${DOCKER_COMMAND} ansible-galaxy install --force --roles-path $(ROLES_PATH) -r $(REQUIREMENTS_FILES)
+
+check: ssh-config
+	@${DOCKER_COMMAND} ansible-playbook --diff --check ${o} $(PLAYBOOK_FILE)
```
Comment (kheraud) [inline makefiles/ansible.mk:37]: Sure it works even if `o` is undefined ?

**Review by kheraud — CHANGES_REQUESTED** [review]
Add a `shell` (/bin/sh) target to all makefile to be able to chroot in docker context

File: makefiles/ansible.mk (hunk 1 above)
Comment (pcarles) [reply makefiles/ansible.mk:37]: Yep if `o` is undefined it is just replaced by a blank string

**Comment by pcarles** [comment]
> Add a `shell` (/bin/sh) target to all makefile to be able to chroot in docker context

 @kheraud  Added in 649caec

**Review by kheraud — APPROVED** [review]
👍🏾 , please do not force-push or rebase after at least one external review, it breaks github "show changes since last review" as commits history is lost

**Pull request merged**